airflow/docker.go

```diff
@@ -813,6 +841,20 @@ func upgradeDockerfile(oldDockerfilePath, newDockerfilePath, newTag, newImage st
 	return nil
 }
 
+func isRuntimeVersion(versionStr string) (bool, error) {
+	// Parse the version string
+	v, err := semver.NewVersion(versionStr)
+	if err != nil {
+		return false, err
+	}
+
+	// Create a reference version 4.0.0
+	referenceVersion := semver.MustParse("4.0.0")
```

Comment:
nit: might want to leave a comment on why version greater than 4.0.0 is consider runtime version